Vokal-Swift.xctemplate/ReuseIdentifierHaver.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  ___FILENAME___
+//  ___PACKAGENAME___
+//
+//  Created by ___FULLUSERNAME___ on ___DATE___.
+//___COPYRIGHT___
+//
+
+import UIKit
+
+/**
+ * Mix-in protocol for cells and views that make use of a reuse identifier. The default implementation
+ * of defaultReuseIdentifier() returns the class name. Add this protocol to a class to use that
+ * default functionality.
+ */
+protocol ReuseIdentifierHaver: class {
```

Comment:
The `Haver` part in the name seems a bit off. Maybe `ReuseIdentifiable` or something similar?

From the Swift API Design Guidelines: https://swift.org/documentation/api-design-guidelines/

> Protocols that describe what something is should read as nouns (e.g. Collection).

> Protocols that describe a capability should be named using the suffixes able, ible, or ing (e.g. Equatable, ProgressReporting).